Dockerfile

```
@@ -20,6 +20,7 @@ ENV POETRY_CACHE_DIR=/tmp/pypoetry
 
 # Required for arm64, for building psutil
 RUN apt-get update && apt-get install --no-install-recommends gcc=4:12.2.0-3 -y && rm -rf /var/lib/apt/lists/*
+RUN apt-get install --no-install-recommends libpq-dev python-dev build-essential libsnappy-dev
 
 RUN pip --no-cache-dir install poetry==1.6.1
```

```suggestion

RUN pip --no-cache-dir install poetry==1.8.2
```
Not sure why I didn't notice this earlier, but this is an old version of poetry.

Try reverting the extra `apt-get` commands and seeing if this by itself fixes it.